tests/Latte/macros.include.escape.phpt

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Test: Latte\Engine: include block escaping
+ */
+
+use Tester\Assert;
+
+
+require __DIR__ . '/../bootstrap.php';
+
+
+$latte = new Latte\Engine;
+$latte->setTempDirectory(TEMP_DIR);
+$latte->setLoader(new \Latte\Loaders\StringLoader());
+
+Assert::error(function () use ($latte) {
+	$latte->renderToString('<meta content="{include #foo}">{block #foo}{$value}{/block}', array('value' => 'b"ar'));
+}, E_USER_WARNING, 'Incompatible context for include, use either |escape or |noescape modifier');
+
```

Comment:
I don't think it as a bypass. Latte just does not know how to handle it properly, so it leaves the decision to the developer.